crates/ruff_python_formatter/src/string/normalize.rs

```diff
@@ -409,7 +418,7 @@ pub(crate) fn normalize_string(
     let mut chars = input.char_indices().peekable();
 
     let is_raw = prefix.is_raw_string();
-    let is_fstring = prefix.is_fstring();
+    let is_fstring = !format_fstring && prefix.is_fstring();
```

Comment:
Shouldn't this be `format_fstring && ...` as per the variable name? Or an I misreading the condition?

Reply:
We could rename the variable. but the negation is intentional. We don't want to run the f-string code if `format_fstring` is enabled.

Reply:
Oh I see. I think it was a mis-understanding from my side. This seems good 👍